src/scripts/configure.sh

```diff
@@ -7,8 +7,9 @@ fi
 
 ORB_ENV_ACCESS_KEY_ID=$(circleci env subst "\$$ORB_ENV_ACCESS_KEY_ID")
 ORB_ENV_SECRET_ACCESS_KEY=$(circleci env subst "\$$ORB_ENV_SECRET_ACCESS_KEY")
```

Comment:
Any particular reason to use envsubst instead of `VALUE=${!ORB_ENV_[NAME]}` for these `ORB_ENV` variables?

Reply:
This has to be POSIX compliant so expanding with `!` will not work